doc/devel/content_guide.rst

```diff
@@ -0,0 +1,273 @@
+
+The plot type gallery displays a selection of the common visualization techniques that
+are implemented in Matplotlib. The gallery is heavily curated and tightly scoped to the
+plotting methods on `matplotlib.axes.Axes` so additions are generally discouraged.
+
+Format
+------
+* very short: 5-10 lines
+* self explanatory data
+* little to no customization.
+
+
+.. _content-user-guide:
+
+User guide
+==========
+
+The goal of the user guide is to explain how Matplotlib works to someone who is
+unfamiliar with the library. The focus is primarily on the functionality provided by the
+library.
+
+Purpose
+-------
+
+The aim of the user guide is to teach the conceptual abstractions on which the
+Matplotlib API is developed so that users can understand how to fit the individual
+components of the library together. Therefore, content should be concept oriented
+rather than focused on specific tasks, e.g.
+
+  Lines in Matplotlib
+
+rather than *How do I make a squiggly yellow line?*
+
+Audience
+--------
+
+The Matplotlib audience encompasses a wide spectrum of users, from readers who are first
+getting introduced to using Matplotlib through the guide to experienced developers who
+want to make something extremely customizied. Instead of trying to write for the
+spectrum, documents should identify their audience so the reader can asses whether the
+document is appropriate for them.
+
+For example, introductory documents should be written with the assumption that
+the reader does not yet know what Matplotlib calls a given visualization task nor
+how any task is accomplished in Matplotlib. Those documents should first introduce or
+define the object/module/concept that it is discussing and why it is important for the
+reader. e.g.
+
+  The `~.Line2D` class is an abstraction of a line and the ``LineCollection`` class is
+  an abstraction of a set of lines. ``Line2D`` and ``LineCollection`` objects manage
+  the properties and behavior of almost every line in an image. This means that one way
+  to modify a line is to call methods on its underlying object.*
+
+Documents that assume familarity with the library should, when possible, be explicit
+about what knowledge the audience is assumed to have, e.g.
+
+
+    This guide assumes that the user understands the general concept of Artists, as
+    explained in :doc:`../users/explain/artists/artist_intro`
+
+Generally the user guide should be written with the assumption that the reader has
+read previous sections and documents should cross reference as necessary rather than
+duplicating information.
+
+
+Scope
+-----
+
+Many concepts in Matplotlib assume a grounding in visualization, statistics, Python
+programming, and other topics to understand how they work. These concepts should be
+contextualized using common terminology, but the focus for all documents should not
+stray from the Matplotlib topic, e.g.
+
+  ``Line2D`` objects take as input a list of pairwise coordinates (x,y) and generates
```

Comment:
This is still at best extremely misleading https://github.com/matplotlib/matplotlib/blob/c287b9f985c3d2231b4c975f8ad7580ce8cbb8ce/lib/matplotlib/lines.py#L287-L308

as `Line2D` takes in an sequence for x and a sequence for y and they are zipped much deeper in the draw process.